### DISCUSSION TITLE
https://www.r-bloggers.com/2024/08/systematic-sampleing-in-r-with-base-r/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#4] [Comments]
# https://www.r-bloggers.com/2024/08/systematic-sampleing-in-r-with-base-r/

Introduction In this post, we will explore systematic sampling in R using base R functions. Systematic sampling is a technique where you select every (k^{th}) element from a list or dataset. This method is straightforward and useful when you wa...

https://www.r-bloggers.com/2024/08/systematic-sampleing-in-r-with-base-r/

**Comment by djmangen** [comment]
Useful yes -- but just to be clear should in no way be confused with random sampling.

**Reply by spsanderson** [reply]
You are correct, hence the title systematic.
